CODEOWNERS

```diff
@@ -219,9 +219,9 @@
 /bundles/org.openhab.binding.mpd/ @stefanroellin
 /bundles/org.openhab.binding.mqtt/ @openhab/add-ons-maintainers
 /bundles/org.openhab.binding.mqtt.espmilighthub/ @Skinah
-/bundles/org.openhab.binding.mqtt.generic/ @openhab/add-ons-maintainers
-/bundles/org.openhab.binding.mqtt.homeassistant/ @antroids
-/bundles/org.openhab.binding.mqtt.homie/ @openhab/add-ons-maintainers
+/bundles/org.openhab.binding.mqtt.generic/ @openhab/add-ons-maintainers @ccutrer
+/bundles/org.openhab.binding.mqtt.homeassistant/ @antroids @ccutrer
+/bundles/org.openhab.binding.mqtt.homie/ @openhab/add-ons-maintainers @ccutrer
```

Comment:
For MQTT generic and MQTT homie, in case you would like to become the "official" maintainer, please remove "openhab/add-ons-maintainers"